src/display/api.js

```diff
@@ -604,16 +618,16 @@ class PDFDocumentProxy {
   /**
    * @param {{num: number, gen: number}} ref - The page reference. Must have
    *   the `num` and `gen` properties.
-   * @returns {Promise} A promise that is resolved with the page index (starting
-   *   from zero) that is associated with the reference.
+   * @returns {Promise<{num: number, gen: number}>} A promise that is resolved
+   *   with the page index that is associated with the reference.
```

Comment:
This removed part of the existing documentation, i.e. the parenthesised part; please revert that.

Reply:
reverted